src/app/modals/place-order/place-order.component.html

```diff
@@ -5,7 +5,7 @@
   </p>
   <p class="widget-help">
     Before paying for this order, your bid needs to be accepted by the Seller.<br>
-    After it is accepted, you can pay for the order in Buy > Orders.
+    After it is accepted, you can pay for the order in Buy/Orders.
```

Comment:
This needs to be reverted

Reply:
Possibly use &gt;

Reply:
okay, lint issue to use `>`, I'll ask to @AllienWorks for alternative